### PR TITLE
Ig res partner vk

### PR DESCRIPTION
**Pull request by igallyamov**
1. add vk field
2. validate field
3. autocompletion to https://vk.com/

### CODE REVIEW

File: res_partner_vk/models.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                vk = "https://vk.com/"+vk
+            else:
+                vk = re.sub("^(http|https)://(vkontakte|vk).(ru|com)",
+                            "https://vk.com", vk)
+            vals['vk'] = vk
+        return vals
+
+    @api.one
+    @api.constrains('vk')
+    def _check_something(self):
```
Comment (yelizariev) [inline res_partner_vk/models.py]: choose more clear function name =)

File: res_partner_vk/models.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        regex = r'^https://vk.com/\w+$'
+        if not re.findall(regex, self.vk):
+            raise ValidationError("VK address %s is not valid.\n The address \
+                must be in the format: https://vk.com/id" % self.vk)
```
Comment (yelizariev) [inline res_partner_vk/models.py]: Error message doesn't correspond to test, e.g. vk field can be https://vk.com/mysql

File: res_partner_vk/static/src/js/vk.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+                    .text(this.get('value') || '');
+        }
+    },
+	vk_uri:function(){
+		//https://vk.com/user
+		return this.get('value');
+	},
+});
+
+instance.web.form.widgets.add('vk', 'instance.web.form.FieldVK');
```
Comment (yelizariev) [inline res_partner_vk/static/src/js/vk.js]: There is widget 'url' already. You don't need to create new one

**Comment by yelizariev** [comment]
check my comments

**Comment by igallyamov** [comment]
1. simplified JS code
2. fix error message

**Comment by yelizariev** [comment]
There is widget 'url' already. You don't need to create new one

**Comment by igallyamov** [comment]
simplified code

**Pull request merged**